.github/workflows/docker-tag.yml

```
@@ -62,23 +62,23 @@ jobs:
          sed -i s/$oldappversion/$newappversion/g K8s-deployment/Charts/resource-server/Chart.yaml
          sed -i s/$oldtag4_5_0/$newtag4_5_0/g K8s-deployment/Charts/resource-server/values.yaml
          git add Docker-Swarm-deployment/single-node/resource-server/rs-stack.yaml K8s-deployment/Charts/resource-server/values.yaml K8s-deployment/Charts/resource-server/Chart.yaml
-         git commit --allow-empty -m "updated rs docker image tag to $newtag4_5_0"
-         git push --set-upstream origin rs-4.5.0-automatic-updates/$newtag4_5_0
+         git commit --allow-empty -m "updated rs docker image tag to $newtag5_5_0"
+         git push --set-upstream origin rs-5.5.0-automatic-updates/$newtag5_5_0
 
          
-         # Creates a new pull request on the datakaveri/iudx-deployment repository with the base branch 4.5.0
+         # Creates a new pull request on the datakaveri/iudx-deployment repository with the base branch 5.5.0
 
-         gh pr create -R datakaveri/iudx-deployment --base 4.5.0 --fill 
+         gh pr create -R datakaveri/iudx-deployment --base 5.5.0 --fill 
         fi
-        
-        if [ "$newtag5_0_0" != "$oldtag5_0_0" ]
+
+        if [ "$newtag5_5_1" != "$oldtag5_5_1" ]
         then
          git checkout master
-         git checkout -b rs-automatic-updates/$newtag5_0_0
+         git checkout -b rs-automatic-updates/$newtag5.5.1
```

variable is `newtag5_5_1`
```suggestion
         git checkout -b rs-automatic-updates/$newtag5_5_1
```